jobs/build/update-base-images/build.sh

```diff
@@ -13,15 +13,20 @@ export REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt
 # `rhel7` is root 0`
 # `nodejs6` is 1001`
 
-TAG="rhaos-4.0-rhel-7"
-URL="http://pkgs.devel.redhat.com/cgit/containers/openshift-enterprise-base/plain/.oit/signed.repo?h=${TAG}"
 USER_USERNAME="--user=ocp-build"
 
 build_common() {
     img=$1; from=$2; user=$3
     shift; shift; shift
     TARGET_DIR=build-$img
     rm -rf ${TARGET_DIR}
+    # for RHEL/UBI 8 and RHEL/UBI 7 we use different tags
+    case "$img" in
+        # for RHEL7 rhaos-4.0-rhel-7 is not in use, for RHEL8 rhaos-4.1-rhel-8 is not in use.
+        ubi8) TAG="rhaos-4.1-rhel-8" ;;
+        *) TAG="rhaos-4.0-rhel-7" ;;
+    esac
+    URL="http://pkgs.devel.redhat.com/cgit/containers/openshift-enterprise-base/plain/.oit/signed.repo?h=${TAG}"
```

Comment:
ah, and you get a different repofile for the different branches... brilliant.
however, why call it TAG? i'd rather see it called BRANCH.